modules/standard/raid/loot_lists_controller.py

```diff
@@ -315,17 +404,23 @@ def build_list(self, items, raid=None, category=None, add_all=False):
             comment = " (%s)" % item.comment if item.comment != "" else ""
 
             if self.setting_service.get("use_item_icons").get_value():
-                item_link = self.text.make_item(item.low_id, item.high_id, item.ql, "<img src=rdb://%s>" % item.icon)
+                item_link = self.text.make_item(item.low_id,
+                                                item.high_id if item.ident == 0 else item.ident,
+                                                item.ql,
+                                                self.text.make_image(
+                                                    item.icon if item.forced_icon == 0 else item.forced_icon))
                 blob += "%s\n%s%s\n | %s\n\n" % (item_link, item.name, comment, add_links)
             else:
-                item_link = self.text.make_item(item.low_id, item.high_id, item.ql, item.name)
+                item_link = self.text.make_item(item.low_id, item.high_id if item.ident == 0 else item.ident, item.ql,
+                                                item.name)
+                # item_link = self.text.make_item(item.low_id, item.high_id, item.ql, item.name)
                 blob += "%s%s\n | %s\n\n" % (item_link, comment, add_links)
-
+            last_item = item
         return blob
 
     def get_items(self, raid, category):
         return self.db.query(
-            "SELECT r.raid, r.category, r.id, r.ql, r.name, r.comment, "
+            "SELECT r.raid, r.category, r.ident, r.forced_icon, r.id, r.ql, r.name, r.comment, "
```

Comment:
`r.ident` should be renamed to `r.high_id`, and then aliased in the query:
`r.high_id AS raid_loot_high_id` or something similar so it's more expressive

Reply:
applied in the next commit; its not even needed to select the r.ident anymore, since - thanks to the new join statement from you - it seems to set the id's & icons correctly.

Reply:
ok but we should still rename `ident` to `high_id` in the raid_loot.sql file unless you are going to remove it from there as well